### PR TITLE
Read write 32

### PR DESCRIPTION
**Pull request by eddierichter-amd**
Adding functionality in the driver and runtime to generate read/write 32 packets to perform AIE configuration. In this implementation, the driver uses the first queue, which is mapped to the ARM, and can send read/write 32 packets to the firmware in response to read/write operations to the driver's sysfs interface. Currently still have the AIE BAR in the platform, which is unused but not hurting anything. Will remove that from the platform once we completely deprecate the MMIO configuration.

### CODE REVIEW

File: runtime_lib/controller/main.cpp [hunk 1]
```diff
@@ -61,7 +61,7 @@ int shim_dma_cols[NUM_SHIM_DMAS] = {2,  3,  6,  7,  10, 11, 18, 19,
 int col_dma_cols[NUM_COL_DMAS] = {7, 8, 9, 10};
 #define NUM_DMAS (NUM_SHIM_DMAS + NUM_COL_DMAS)
 
-#define CHATTY 0
+#define CHATTY 1
```
Comment (jgmelber) [inline runtime_lib/controller/main.cpp:64]: CHATTY 0

**Comment by jgmelber** [comment]
What version, if any, of libxaie does this need?

**Comment by eddierichter-amd** [comment]
> What version, if any, of libxaie does this need?

From the host, it needs a libxaie backend which interacts with the driver's sysfs interface. This is implemented in Joel's fork.

**Review by jgmelber — CHANGES_REQUESTED** [review]

File: test/13_mb_add_one/Makefile [hunk 2]
```diff
@@ -9,7 +9,7 @@ all: test.elf
 
 test.elf: aie.mlir test.cpp
 	aiecc.py -v --host-target=x86_64-amd-linux-gnu --sysroot= $< -I/opt/xaiengine/include -L/opt/xaiengine/lib -Wl,-R/opt/xaiengine/lib \
-						-I$(ACDC_AIR)/runtime_lib/airhost/include -I$(ACDC_AIE)/runtime_lib $(ACDC_AIE)/runtime_lib/test_library.cpp \
+						-I$(ACDC_AIR)/runtime_lib/airhost/include -I$(ACDC_AIE)/runtime_lib/x86_64/test_lib/include $(ACDC_AIE)/runtime_lib/x86_64/test_lib/src/test_library.cpp \
```
[the remark this answers is not in the document]
Comment (eddierichter-amd) [reply test/13_mb_add_one/Makefile:12]: Good point. 21 Makefile is updated here [2a56883](https://github.com/Xilinx/mlir-air/pull/248/commits/2a56883f8f751bc85444ed4a220f1b3d496fed95) added the others here [cb2e407](https://github.com/Xilinx/mlir-air/pull/248/commits/cb2e4076d2f1d7021b199dc5e9b35cff2154bf22).

**Comment by eddierichter-amd** [comment]
> Can this be closed in light of #250 ?

Yep, closing now.

**Pull request closed without merging**